src/konch/__init__.py

```diff
@@ -511,7 +511,28 @@ def start(self) -> None:
             raise ShellNotAvailableError("PtPython shell not available.") from error
         print(self.banner)
 
-        config_dir = Path("~/.ptpython/").expanduser()
+        # Determine config directory
+        config_dirs = [
+            Path(os.environ.get("XDG_CONFIG_HOME", "~/.config")) / "ptpython",
```

Comment:
thought(non-blocking): should we use [platformdirs](https://pypi.org/project/platformdirs/), the actively-maintained equivalent of [appdirs](https://github.com/ActiveState/appdirs?tab=readme-ov-file), which ptpython uses to select the config directory?

i don't have a strong opinion either way. the code here is short and understandable enough.

Reply:
platformdirs is tiny and stable enough; i've got no issues including it as a dep here